src/desktop/apps/artist/routes.coffee

```diff
@@ -18,6 +18,7 @@ currentVeniceFeature = require './components/current_venice_feature/index'
 sd = require('sharify').data
 
 @index = (req, res, next) ->
+  console.log('referrer:', req.get('Referrer'))
```

Comment:
Oops bunch of strays left from pairing 👅

Reply:
I knew I would forget at least one lol.